charmhelpers/contrib/hahelpers/apache.py

```diff
@@ -37,8 +38,8 @@
 
 def get_cert(cn=None):
     # TODO: deal with multiple https endpoints via charm config
-    cert = config_get('ssl_cert')
-    key = config_get('ssl_key')
+    cert = base64.b64decode(config_get('ssl_cert'))
```

Comment:
This is a helper function to reliably get the base64 encoded data from config/relations.  It is consumed by the ApacheSSLContext [0] which does the base64 decoding.

0: https://github.com/juju/charm-helpers/blob/9248c1a06dfbdfba21efeddd02af13fd7b58386e/charmhelpers/contrib/openstack/context.py#L959

Reply:
well, according to lp:#1841429 this it doesn't.